scripts/cli/generate_completion_scripts.sh

```diff
@@ -21,7 +21,7 @@
 GIT_ROOT=$(git rev-parse --show-toplevel)
 
 (cd $GIT_ROOT &&
-  ./gradlew :cli:jvmRun --args='--generate-completion=bash' -DmainClass=org.eclipse.apoapsis.ortserver.cli.OrtServerMainKt --quiet > integrations/completions/osc-completion.bash &&
-  ./gradlew :cli:jvmRun --args='--generate-completion=fish' -DmainClass=org.eclipse.apoapsis.ortserver.cli.OrtServerMainKt --quiet > integrations/completions/osc-completion.fish &&
-  ./gradlew :cli:jvmRun --args='--generate-completion=zsh' -DmainClass=org.eclipse.apoapsis.ortserver.cli.OrtServerMainKt --quiet > integrations/completions/osc-completion.zsh
+  ./gradlew --quiet :cli:jvmRun --args='--generate-completion=bash' -DmainClass=org.eclipse.apoapsis.ortserver.cli.OrtServerMainKt > integrations/completions/osc-completion.bash &&
```

Comment:
Hmm, I overlook that `--quiet` was already present at the end. Actually, I believe it should have worked that way, too. Any idea why it didn't / have you verified that it works now as expected when specified at the beginning?

Reply:
> Any idea why it didn't

Unfortunately, I don't know.

> have you verified that it works now as expected when specified at the beginning?

Having the check now succeeding, while it was failing multiple times before, should verify that it works.